definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -71,7 +70,7 @@ config {
         count_of_fails: "Counts the number of Failed outcome states for a participant.",
         cohort_id: "ID of cohort.",
         npq_lead_provider_id: "ID of Lead Provider.",
-        participant_identity_id: "This is the ID associated with the email a participant uses to sign up. As a participant can have multiple emails, this doesn't accurately represent a single profile.",
+        participant_identity_id: "This is the ID associated with the email a participant uses to sign up. This id remains the same across participant's NPQ applications and NPQ participant profile (assuming they've signed up with the same email). As a participant can have multiple emails, this isn't the most accurate representation of a single user. ecf_user_id (or trn_verified) is a much better representaiton of a single user .",

```

Comment:
"**This id will remain the same across a participant's NPQ applications and NPQ participant profile so long as they have signed up with the same email**"